vsintegration/src/FSharp.Editor/Hints/FSharpInlayHintsService.fs

```diff
@@ -4,24 +4,24 @@ namespace Microsoft.VisualStudio.FSharp.Editor.Hints
 
 open System.Collections.Immutable
 open System.ComponentModel.Composition
-open Microsoft.CodeAnalysis.ExternalAccess.FSharp.InlineHints
+open Microsoft.CodeAnalysis.ExternalAccess.FSharp.InlayHints
 open Microsoft.VisualStudio.FSharp.Editor
 open Microsoft.VisualStudio.FSharp.Editor.Telemetry
 open CancellableTasks
 open System.Threading.Tasks
 
-// So the Roslyn interface is called IFSharpInlineHintsService
+// So the Roslyn interface is called IFSharpInlayHintsService
 // but our implementation is called just HintsService.
-// That's because we'll likely use this API for things other than inline hints,
+// That's because we'll likely use this API for things other than inlay hints,
 // e.g. signature hints above the line, pipeline hints on the side and so on.
 
-[<Export(typeof<IFSharpInlineHintsService>)>]
-type internal FSharpInlineHintsService [<ImportingConstructor>] (settings: EditorOptions) =
+[<Export(typeof<IFSharpInlayHintsService>)>]
```

Comment:
I would imagine this should stay the same, since it's what roslyn uses to import it.

Reply:
I guess the minimum working change would be to change the naming in the messages and options. Pretty much everything non-code, and what has "inline" as a string.